README.md

```diff
@@ -1 +1,40 @@
-# School_Library
\ No newline at end of file
+# School_Library
+
+## Built With
+ - Ruby
+ - Linters: Rubocop
+ - Visual Studio Code
+
+# Getting Started
+To get a local copy up and running follow these simple steps.
+
+## Prerequisites
+ A web browser like Google Chrome.
+ A code editor like Visual Studio Code with Git and Ruby.
+You can check if Git is installed by running the following command in the terminal.
+
+$ git --version
+Likewise for Ruby installation.
+
+$ ruby --version && irb
+## Setup
+Clone the repository using the GitHub link provided below.
+
+## Install
+In the terminal, go to your file directory and run this command.
+
+$ git clone https://github.com/aneallaryea100/School_Library.git
+
+Usage 
+Kindly modify the files as needed.
+
+## Authors
+👤 Aneal Laryea
+
+- GitHub: @aneallaryea100
+- Twitter: @AnealLaryea
+- LinkedIn: (https://www.linkedin.com/in/nii-aneal-84ba7a147/?locale=fr_FR)
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] Kindly add a license file to your project using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: because a license is very important for your project :thinking:
- [ ] Also, it would be nice if you add your license file link here in the readme file so it would be easily accessible for all the viewers and will make your readme file descriptive :blush: as shown below :arrow_down:

```
### License :memo:

This project is [MIT](./LICENSE) licensed.
```